core/web/resolver/p2p.go

```diff
@@ -0,0 +1,86 @@
+package resolver
+
+import (
+	"github.com/graph-gophers/graphql-go"
+
+	"github.com/smartcontractkit/chainlink/core/services/keystore/keys/p2pkey"
+)
+
+type P2PKey struct {
```

Comment:
```suggestion
type P2PKeyResolver struct {
```

Also make a constructor for this

Reply:
Fixed!